tsconfig.json

```diff
@@ -12,6 +12,5 @@
     "jsx": "react",
     "resolveJsonModule": true
   },
-  "lib": ["esnext"],
```

Comment:
Why this change?

Reply:
It's not a valid field at all, it's actually in compilerOptions (which has the same value)

Reply:
🤦